### PR TITLE
Simpl Payments Icon

### PR DESCRIPTION
**Pull request by mrinalika-ghosh**
## Why are you adding this icons?
I'm adding/updating this icon(s) because Simpl Payments offers Pay Later & Pay in 3 solutions to customers in India (https://getsimpl.com/). This is a payment method offered on Shopify via Simpl Payments App today
#### Help us identify yourself
- [x] I'm working/collaborating with the brand directly and they have provided the icons.
- [ ] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.
Link to the brand guidelines: <Link>
## Checklist to add new icons
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I have attached a screenshot comparison with the example icon provided in guidelines
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.
If this pull request is not adding new icons, you can remove this checklist.
## Attach a screenshot of the icon along side the example Visa icon
screenshot
#### Tips how to create a screenshot
We have found free online SVG editor https://www.freecodeformat.com/svg-editor.php very useful to create one. Here is a sample code for you to verify that you icon appears properly along side the placeholder.
screenshot
```
<!-- Change background color if needed to showcase your icon better -->
<style> body { background: white; } </style>
<!-- DO NOT DELETE EXAMPLE -->
<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-visa"><title id="pi-visa">Visa</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M28.3 10.1H28c-.4 1-.7 1.5-1 3h1.9c-.3-1.5-.3-2.2-.6-3zm2.9 5.9h-1.7c-.1 0-.1 0-.2-.1l-.2-.9-.1-.2h-2.4c-.1 0-.2 0-.2.2l-.3.9c0 .1-.1.1-.1.1h-2.1l.2-.5L27 8.7c0-.5.3-.7.8-.7h1.5c.1 0 .2 0 .2.2l1.4 6.5c.1.4.2.7.2 1.1.1.1.1.1.1.2zm-13.4-.3l.4-1.8c.1 0 .2.1.2.1.7.3 1.4.5 2.1.4.2 0 .5-.1.7-.2.5-.2.5-.7.1-1.1-.2-.2-.5-.3-.8-.5-.4-.2-.8-.4-1.1-.7-1.2-1-.8-2.4-.1-3.1.6-.4.9-.8 1.7-.8 1.2 0 2.5 0 3.1.2h.1c-.1.6-.2 1.1-.4 1.7-.5-.2-1-.4-1.5-.4-.3 0-.6 0-.9.1-.2 0-.3.1-.4.2-.2.2-.2.5 0 .7l.5.4c.4.2.8.4 1.1.6.5.3 1 .8 1.1 1.4.2.9-.1 1.7-.9 2.3-.5.4-.7.6-1.4.6-1.4 0-2.5.1-3.4-.2-.1.2-.1.2-.2.1zm-3.5.3c.1-.7.1-.7.2-1 .5-2.2 1-4.5 1.4-6.7.1-.2.1-.3.3-.3H18c-.2 1.2-.4 2.1-.7 3.2-.3 1.5-.6 3-1 4.5 0 .2-.1.2-.3.2M5 8.2c0-.1.2-.2.3-.2h3.4c.5 0 .9.3 1 .8l.9 4.4c0 .1 0 .1.1.2 0-.1.1-.1.1-.1l2.1-5.1c-.1-.1 0-.2.1-.2h2.1c0 .1 0 .1-.1.2l-3.1 7.3c-.1.2-.1.3-.2.4-.1.1-.3 0-.5 0H9.7c-.1 0-.2 0-.2-.2L7.9 9.5c-.2-.2-.5-.5-.9-.6-.6-.3-1.7-.5-1.9-.5L5 8.2z" fill="#142688"/></svg>
<!-- TODO: insert your icon here -->
<YOUR SVG CODE>
<br>
<!-- TODO: insert your icon here -->
<YOUR SVG CODE>
</br
```
#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)
Shopify India Team as a Payment Partner
What's the expected date of this change to deploy on Shopify?

### CODE REVIEW

**Comment by hellicarusprime** [comment]
Hi @mrinalika-ghosh, the icon still isn't the right size, but it looks like I accidentally missed the link to our [contributing guidelines](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) in my last message. Refer to these to ensure your icon is correct.

**Comment by mrinalika-ghosh** [comment]
Hi Michael, We have updated again the file. Can you please check the latest svg file now

**Comment by hellicarusprime** [comment]
Hi @Mrinalika, the size of the icon you added is 40 x 26—it should be 38 x 24. The visible border should be included.

**Comment by mrinalika-ghosh** [comment]
@hellicarusprime - Updated

**Comment by hellicarusprime** [comment]
Hi @mrinalika-ghosh, there are still a couple of issues with the icon. Please refer to our [contributing guidelines](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) to address these.

Issues:
- The name of the SVG must be the same as the name entered in db/payment_icons.yml
- The border must have a 2px radius (outside stroke)

**Comment by mrinalika-ghosh** [comment]
@hellicarusprime Apologies for the back and forth. Hope this time we were able to make the required changes.

**Review by hellicarusprime — APPROVED** [review]

**Comment by hellicarusprime** [comment]
The icon looks good now @mrinalika-ghosh, but you have some conflicts that need to be resolved.

**Comment by mrinalika-ghosh** [comment]
Hey! Thanks, @hellicarusprime  - are there any other tasks for me to complete?
Also, when will this show up in the Shopify Payment Icons as an option?

**Comment by mrinalika-ghosh** [comment]
@hellicarusprime Just wanted to check since the test has failed. Are we supposed to make any changes?

**Comment by hellicarusprime** [comment]
Cc: @adeniyiao

**Comment by hellicarusprime** [comment]
> @hellicarusprime Just wanted to check since the test has failed. Are we supposed to make any changes?

Cc: @adeniyiao

**Comment by mrinalika-ghosh** [comment]
Have resolved the conflicts again @hellicarusprime Let me know in case we are suppose to update anything else from our end

File: app/assets/images/payment_icons/simpl.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Lovedanihonjin) [inline app/assets/images/payment_icons/simpl.svg:1]: The CI checks failed due to incomplete markup.  
`{:message=>"The 'simpl' SVG file should have a single <title> tag"}.`

Please make sure that your SVG markup meets the following [criteria](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#guidelines-for-new-icons).

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Requested changes on SVG markup

**Comment by mrinalika-ghosh** [comment]
@Lovedanihonjin : Can you please check once now?

**Review by Lovedanihonjin — APPROVED** [review]

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
CI check resulted in this error: {:message=>"The 'clip0_3721_39148' ID should be pi-simpl-clip0_3721_39148 (missing 'pi-' prefix)"}.
Expected /pi-(.*)/ to match "clip0_3721_39148".

Please resolve, thank you!

**Comment by mrinalika-ghosh** [comment]
Resolved the above error

**Comment by mrinalika-ghosh** [comment]
@Lovedanihonjin : Let me know if still I am missing anything in my latest commit?

**Review by Lovedanihonjin — CHANGES_REQUESTED** [review]
Looks great, but the CI check resulted in this error `The 'simpl' SVG file should contain a single line of markup, optionally terminated by an empty line`

Please resolve. Thank you!

**Comment by mrinalika-ghosh** [comment]
@Lovedanihonjin : Have made the changes again. Let me know if anything else needed from my end.

**Review by Lovedanihonjin — APPROVED** [review]
Looks great, thanks so much! Next deploy is on June 5.

**Pull request merged**